src/renderer/components/Wallet/WalletTable.vue

```diff
@@ -212,7 +212,7 @@ export default {
     },
 
     delegateName (row) {
-      return row.votedDelegate ? row.votedDelegate.username : ''
+      return row.vote ? (this.$store.getters['delegate/byPublicKey'](row.vote)).username : ''
```

Comment:
Can you change to the `if` condition? to make it more readable:

```suggestion
      if (row.vote) {
        const delegate = this.$store.getters['delegate/byPublicKey'](row.vote)
        return delegate.username
      }
      return ''      
```